executor/batch_point_get.go

```diff
@@ -162,6 +166,39 @@ func (e *BatchPointGetExec) Open(context.Context) error {
 	return nil
 }
 
+// CacheTable always use memBuffer in session as snapshot.
+// cacheTableSnapshot inherits kv.Snapshot and override the BatchGet methods to return empty.
+type cacheTableSnapshot struct {
+	kv.Snapshot
+	memBuffer kv.MemBuffer
+}
+
+func (s cacheTableSnapshot) BatchGet(ctx context.Context, keys []kv.Key) (map[string][]byte, error) {
```

Comment:
`Get` is not implemented. I think you can add some unit tests for `cacheTableSnapshot`

Reply:
okay